nats.go

```diff
@@ -266,8 +269,15 @@ type Options struct {
 
 	// DisconnectedCB sets the disconnected handler that is called
 	// whenever the connection is disconnected.
+	// Will not be called if DisconnectedErrCB was initiated.
+	// DEPRECATED. use DisconnectedErrCB which passes initial err
```

Comment:
`which passes error that caused the disconnect event`?

Reply:
thanks, fixed docs